Only search packages from tur-pypi repository when auto-updating.
--- a/scripts/bin/update-packages
+++ b/scripts/bin/update-packages
@@ -189,7 +189,7 @@
 echo "INFO: Running update for: $*"
 
 if [[ "$1" == "@all" ]]; then
-	for repo_dir in "tur" "tur-on-device"; do
+	for repo_dir in "tur-pypi"; do
 		for pkg_dir in "${repo_dir}"/*; do
 			! _should_update "${pkg_dir}" && continue # Skip if not needed.
 			# Update all its dependencies first.
@@ -206,7 +206,7 @@
 else
 	for pkg in "$@"; do
 		if [ ! -d "${pkg}" ]; then # If only package name is given, try to find it's directory.
-			for repo_dir in "tur" "tur-on-device"; do
+			for repo_dir in "tur-pypi"; do
 				if [ -d "${repo_dir}/${pkg}" ]; then
 					pkg="${repo_dir}/${pkg}"
 					break
